tests/phpunit/tests/rest-api/rest-schema-setup.php

```diff
@@ -134,6 +134,8 @@ public function test_expected_routes_in_schema() {
 			'/wp/v2/comments',
 			'/wp/v2/comments/(?P<id>[\\d]+)',
 			'/wp/v2/global-styles/(?P<id>[\/\w-]+)',
+			'/wp/v2/global-styles/(?P<id>[\d]+)/autosaves',
+			'/wp/v2/global-styles/(?P<parent>[\d]+)/autosaves/(?P<id>[\d]+)',
```

Comment:
@jonnynews

It would be handy to be able to opt-out of these as well if folks want to skip registering these routes, e.g., `'autosave_rest_controller_class'  => false,` in `register_post_type()`.

 https://github.com/WordPress/wordpress-develop/commit/1f51e1f4f6c81238fffed706ff165795ea34d522#diff-52c8c412194f4e6bf199f6bbf576b66410eb470aa181330bdd7c290c9ed075e4R929

Reply:
Or, maybe better, a `supports_autosaves`, similar to `supports_revisions`?

Reply:
@ramonjd For now, we could just add the global post type here - https://github.com/WordPress/wordpress-develop/blob/235cb398918c86b6007e7f37e1daf3a07c681f77/src/wp-includes/class-wp-post-type.php#L925-L927